app/models/chart_data_values.rb

```diff
@@ -159,6 +159,16 @@ def self.as_chart_json(output)
 
 private
 
+  def translate_x_axis_categories
+    return @chart[:x_axis] unless @chart[:chart1_type] == :column
+
+    @chart[:x_axis].map do |date|
+      ApplicationController.helpers.nice_dates(Date.strptime(date, '%d %b %Y'))
```

Comment:
use `Date.strptime(date, '%d %b %Y').to_s(:es_compact)` when https://github.com/Energy-Sparks/energy-sparks/pull/1959/files is merged